storerCassandra/storerCassandra.go

```diff
@@ -432,3 +443,23 @@ func (s StorerCassandra) GetResult(id string) (*storerGeneric.Result, error) {
 
 	return result, err
 }
+
+func (s StorerCassandra) StoreConfig(config *storerGeneric.Config) error {
```

Comment:
I would like a bit more of a validation check here. This might be a good justification for JSON over INI.